### PR TITLE
#169380488 added five day weather forecast

### PR DESCRIPTION
**Pull request by hezzie**
#### What does this PR do?
- Five day weather forecast
#### Description of Task to be completed?
- This PR adds five day weather forecast feature to the app
-  Corrected the folder structure by creating a helper class in a different folder and also merged img and images folder
#### How should this be manually tested?
- **`git clone ft-five-day-forecast-169380488`** 
- **`npm install`**
- **`npm start`** 
#### Any background context you want to provide?
`N/A`
#### What are the relevant pivotal tracker stories?
- [#169380488](https://www.pivotaltracker.com/n/projects/2406895/stories/169380488)
#### Screenshots (if appropriate)
Screen Shot 2019-10-26 at 19 17 08

#### Questions:
`N/A`

### CODE REVIEW

File: src/components/Details.js [hunk 1]
```diff
@@ -1,47 +1,54 @@
-import React, { Component } from 'react';
-import convertDegrees from '../helpers';
+import React, { Component } from "react";
+import Helper from "../helpers/Helpers";
```
Comment (NiyoEric) [inline src/components/Details.js]: Why do you have a helper file inside a helper directory? I think you should change to another name.

File: src/helpers/Helpers.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class Helpers {
```
Comment (NiyoEric) [inline src/helpers/Helpers.js]: @hezzie We can not have `helper` class inside `Helpers` file inside `helper` directory.

**Review by hezzie — COMMENTED** [review]
1. I have added the five day weather forecast in a different branch. 
2. I also managed to map through the array of data instead of hard coding it as i previously did. 
3. In addition to that I stored the icon urls in the .env files and created a .env.example file

**Review by victorkarangwa — APPROVED** [review]

File: src/helpers/Helpers.js (hunk 2 above)
Comment (hezzie) [reply src/helpers/Helpers.js]: I have changed both, thanks.

File: src/components/Details.js (hunk 1 above)
Comment (hezzie) [reply src/components/Details.js]: @NiyoEric  I have changed the name to Converters, thanks for the feedback.

**Review by NiyoEric — APPROVED** [review]

**Comment by ivymwende** [comment]
Good job Hezron! All feedback was updated and this looks good to be merged.

**Review by ivymwende — APPROVED** [review]

**Review by WillyBoris — APPROVED** [review]

**Pull request merged**